### ISSUE TITLE
Android 13 help for a dummy

### ISSUE THREAD

**Issue by DollSkies** [ZaneYork/SMAPI-Android-Installer#328]
I'm on Android 13 and I'm seriously lost on how I'm supposed to get com.zane into the data folder. I have tried every fill manager that everyone has commented, even fv explorer, and I still can't acess my data folder. Any help?

**Comment by ArthurWarden** [comment]
Have you tried zarchiver?

**Comment by Rubb-le** [comment]
I'm not sure if everyone did it this way, but as an Android 13 user, here's a summary of how I did it:

Use whatever file manager you want to extract your mods (I used ZArchiver because that's what I'm used to) and THEN use another file manager like https://play.google.com/store/apps/details?id=com.marc.files to transfer them to the data folder. I know it's messy, but it works. 
Marc files manager allows split screen option which is how you can transfer files. This article shows you how to do it https://droidwin.com/how-to-access-android-13-data-and-obb-folder-transfer-files-in-them/

(Also, in the article they say to press the icon to show the option of split screen, which left me confused for a while. You actually have to long press the the tab itself in recent tabs to show the option.)

I hope this helped :) sorry if it's confusing.

**Comment by DollSkies** [comment]
> I'm not sure if everyone did it this way, but as an Android 13 user, here's a summary of how I did it:
> 
> Use whatever file manager you want to extract your mods (I used ZArchiver because that's what I'm used to) and THEN use another file manager like https://play.google.com/store/apps/details?id=com.marc.files to transfer them to the data folder. I know it's messy, but it works. Marc files manager allows split screen option which is how you can transfer files. This article shows you how to do it https://droidwin.com/how-to-access-android-13-data-and-obb-folder-transfer-files-in-them/
> 
> (Also, in the article they say to press the icon to show the option of split screen, which left me confused for a while. You actually have to long press the the tab itself in recent tabs to show the option.)
> 
> I hope this helped :) sorry if it's confusing.

I will try this when I get the chance, but is this a permanent thing? Or is it only for the alpha build?

**Comment by Rubb-le** [comment]
> > I'm not sure if everyone did it this way, but as an Android 13 user, here's a summary of how I did it:
> > Use whatever file manager you want to extract your mods (I used ZArchiver because that's what I'm used to) and THEN use another file manager like https://play.google.com/store/apps/details?id=com.marc.files to transfer them to the data folder. I know it's messy, but it works. Marc files manager allows split screen option which is how you can transfer files. This article shows you how to do it https://droidwin.com/how-to-access-android-13-data-and-obb-folder-transfer-files-in-them/
> > (Also, in the article they say to press the icon to show the option of split screen, which left me confused for a while. You actually have to long press the the tab itself in recent tabs to show the option.)
> > I hope this helped :) sorry if it's confusing.
> 
> I will try this when I get the chance, but is this a permanent thing? Or is it only for the alpha build?

I'm sure we won't have to do it this way forever 😃 I mean, I hope not.

**Comment by DollSkies** [comment]
> Have you tried zarchiver?

Zarchiver is my baby but I can't edit data files on it for some reason.

**Comment by Roshan9k** [comment]
> I'm on Android 13 and I'm seriously lost on how I'm supposed to get com.zane into the data folder. I have tried every fill manager that everyone has commented, even fv explorer, and I still can't acess my data folder. Any help?

To Everyone Who's Still Having Trouble Installing
• Read Description •
https://youtu.be/PAb-U5Ww-qo

**Comment by Insan23** [comment]
> > Have you tried zarchiver?
> 
> Zarchiver is my baby but I can't edit data files on it for some reason.

Actually, you can access Android/data folder using your phone original file manager
Other file manager won't allow access to Android/data folder for android version 10 and above (some since android 11)


Or you can access it from computer using MTP

**Comment by ArthurWarden** [comment]
> > > Have you tried zarchiver?
> > 
> > 
> > Zarchiver is my baby but I can't edit data files on it for some reason.
> 
> Actually, you can access Android/data folder using your phone original file manager Other file manager won't allow access to Android/data folder for android version 10 and above (some since android 11)
> 
> Or you can access it from computer using MTP

I've got android 10 and both original file manager and zarchiver work fine.

**Comment by Insan23** [comment]
> 
> I've got android 10 and both original file manager and zarchiver work fine.

Yes, my mistake, android 10 still able to access Android/data directory using third party file manager app, and new Android storage policy is in effect since android 11
but you really need to read the author question, which is stating that he is using Android 13
so you can give more precise answer

**Comment by ArthurWarden** [comment]
> > I've got android 10 and both original file manager and zarchiver work fine.
> 
> Yes, my mistake, android 10 still able to access Android/data directory using third party file manager app, and new Android storage policy is in effect since android 11 but you really need to read the author question, which is stating that he is using Android 13 so you can give more precise answer

For this very reason I'm not upgrading to 11 and up. As for the authors question, the only advice i can give anymore is trying "total commander " if this doesn’t help i don’t know what will.

**Comment by DollSkies** [comment]
> > I'm on Android 13 and I'm seriously lost on how I'm supposed to get com.zane into the data folder. I have tried every fill manager that everyone has commented, even fv explorer, and I still can't acess my data folder. Any help?
> 
> To Everyone Who's Still Having Trouble Installing Smapi Alpha Here's Video I Made. • Read Description • https://youtu.be/PAb-U5Ww-qo

Thank you so much! This video was what helped me figure it out

**Comment by bearss69** [comment]
Do you guys have any solution for me? I use Android 13, downloaded all the latest apps, and even purchased stardew valley and yet, after downloading SMAPI SV, the app would always crash every time I open it.

**Comment by luigimannoni** [comment]
> > I'm on Android 13 and I'm seriously lost on how I'm supposed to get com.zane into the data folder. I have tried every fill manager that everyone has commented, even fv explorer, and I still can't acess my data folder. Any help?
> 
> To Everyone Who's Still Having Trouble Installing Smapi Alpha Here's Video I Made.
> • Read Description •
> https://youtu.be/PAb-U5Ww-qo

This video has been deleted for YouTube breach, can you explain in short points what are the exact steps? Thanks
